gk/main.c

```diff
@@ -582,9 +723,9 @@ gk_proc(void *arg)
 }
 
 struct gk_config *
-alloc_gk_conf(void)
+get_gk_conf(void)
```

Comment:
Good suggestion. So, we can realize the dependencies among blocks through passing parameters, just as we did for ``net`` configuration.